### DISCUSSION TITLE
Can't detect mpd

### DISCUSSION THREAD

**Discussion by antingudud** [jcorporation/myMPD#671] [Q&A] [answered]
Started mympd but it's stuck in
```
Connecting to MPD socket "/run/mpd/socket"
MPD connection: No such file or directory
MPD connection failed
```
it seems like it can't detect mpd, also there's nothing in /run/mpd/. Is it because my mpd config folder is in home directory under .config and activated by systemd with user setting?
my [mpd.conf](https://pastebin.com/VTm1Am7f) and 
```
/home/urameshiya/.config/mympd/config/http_host
localhost
```
```
/home/urameshiya/.config/mympd/config/http_port
8000
```
if that'll help

**Comment by jcorporation** [comment, marked as answer]
You can configure the mpd connection in the webui.
